script/blog.js

```diff
@@ -0,0 +1,66 @@
+const headOne = document.createElement('h2');
+headOne.textContent = 'Adoptadog Blog';
+document.getElementById("container").appendChild(headOne);
+
+const i = [{
+		img: "img/blog-1.jpg",
+		h3:'Traveling With Your Dog',
+		p1: 'typesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industry',
+		p2: 'tTttypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industry'
+},
+{
+		img: "img/blog-2.jpg",
+		h3:'Traveling With Your Dog2test',
+		p1: 'typesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industry',
+		p2: 'tTttypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industry'
+},
+{
+		img: "img/blog-3.jpg",
+		h3:'Traveling With Your Dog3test',
+		p1: 'typesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industry',
+		p2: 'tTttypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industrytypesetting industry. Lorem Ipsum has been the industry'
+}
+];
+
+for (let z = 0; z < i.length; z+=1){
```

Comment:
If you're using a non semantic variable for your counter you should use i. It's kind of a best practice. also, you can shorten the incrementer to z++